spec/factories/item.rb

```diff
@@ -3,5 +3,13 @@
     title "item name"
     description "yummy"
     price        5
+
+
+    factory :item_with_categories do
+       after(:create) do |item, evaluator|
+        category = create(:category)
+        item.item_categories.create(category: category)
+      end
```

Comment:
`end` at 12, 6 is not aligned with `after(:create) do |item, evaluator|` at 9, 7